### PR TITLE
The "Send" Button in the Footer Section Should Be Centered #464

### PR DESCRIPTION
**Pull request by arpit74170**
I have done the fixing of the send button n the center of the form as request by the user

# Pull Request Format
## Type of PR
- **Add X in the box to specify the improvement type.**
- [X] Bug fix
- [x] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): ___________  

## Description
This pull request aims to [briefly describe what the pull request does, e.g., "add a new feature," "fix a bug," "improve documentation," etc.]. 

## Screenshots / Videos (if applicable)
**Before:**
- [Describe the state before the changes, e.g., "The section headings in the footer were misaligned, leading to an inconsistent and cluttered appearance."]

**After:**
- [Describe the state after the changes, e.g., "The section headings in the footer are now properly aligned, providing a clean and professional look."]

## Checklist
- **Add X in the box to specify.**
- [X] I have performed a self-review of my code.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

### Additional Context
[Provide any additional context about the changes, such as specific challenges you faced or decisions you made. This can help reviewers understand the rationale behind your changes.]

Thank you for reviewing my pull request!

### CODE REVIEW

**Comment by arpit74170** [comment]
Look on it...if it is ok then merge it...otherwise tells me what is not okk

File: index.html [hunk 1]
```diff
@@ -675,7 +675,7 @@ <h2>Feedback Form</h2>
                     <input type="email" name="email" class="text-input contact-input"
                         placeholder="Your email address...">
                     <textarea name="message" class="text-input contact-input" placeholder="Your message..."></textarea>
-                    <button type="submit" class="btn btn-big contact-btn">
+                    <button style="position: relative ; left: -50px" type="submit" class="btn btn-big contact-btn ">
```
Comment (swaraj-das) [inline index.html:678]: Don't add styles directly in the HTML. We have a CSS file for that. Mixing them makes the project harder to maintain.

**Comment by arpit74170** [comment]
Okay i will fix it.. today only...dont assign to any one else

File: index.html (hunk 1 above)
Comment (arpit74170) [reply index.html:678]: Okay i will fix it today only....dont assign to any one else

**Comment by arpit74170** [comment]
Check it and give me response

**Comment by arpit74170** [comment]
Sir pls have a look on it

**Comment by arpit74170** [comment]
I am adding new pull reuest

**Pull request closed without merging**